runtests.sh

```diff
@@ -92,13 +94,18 @@ if [ ! -z "$RUN_INTEGRATION" ]; then
     run_integration
 fi
 
-if [ ! -z "$RUN_EXAMPLES" ]; then
+if [ ! -z "$RUN_DEMOS" ]; then
     if [ "$1" == "examples" ] ; then
         # shift to remove the test suite name and be able to pass the rest
         # to the examples suite.
         shift
     fi
-    run_examples "$@"
+    if [ "$1" == "demos" ] ; then
+        # shift to remove the test suite name and be able to pass the rest
+        # to the examples suite.
```

Comment:
Good catch that I didn't change the comment! (proof I didn't do a global sed on purpose :))